src/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ include_directories(${ICU_INCLUDE_DIRS})
 include_directories(${ICONV_INCLUDE_DIRS})
 include_directories(${DECNUMBER_INCLUDE_DIR})
 include_directories(${EXTRA_CORE_INCLUDE_DIRS})
+include_directories(SYSTEM ${LIBUNWIND_INCLUDE_DIR})
```

Comment:
Why `SYSTEM`?

AFAIU you drop `-DCMAKE_FIND_USE_CMAKE_SYSTEM_PATH=FALSE` from the static build because of this, but it's unclear why you need it.

Reply:
https://cmake.org/cmake/help/latest/command/include_directories.html:

> If the `SYSTEM` option is given, the compiler will be told the directories are meant as system include directories on some platforms. Signalling this setting might achieve effects such as the compiler skipping warnings, or these fixed-install system files not being considered in dependency calculations - see compiler docs.

I assume we can treat libunwind headers as system ones — otherwise we will get compiler warnings, which are treated as errors.

Reply:
> AFAIU you drop -DCMAKE_FIND_USE_CMAKE_SYSTEM_PATH=FALSE from the static build because of this, but it's unclear why you need it.

Nope, I dropped it because otherwise CMake is not able to find _libunwind.h_ on _macOS_.
Please see point 6 of search procedure: https://cmake.org/cmake/help/git-stage/command/find_file.html#command:find_file.